packages/gestalt-design-tokens/tokens/vr-theme/$metadata.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
is order required by some reason?

Reply:
This is generated by token-studio, I am not quite sure why